app/src/main/java/tech/relaycorp/courier/data/network/CogRPC.kt

```diff
@@ -0,0 +1,26 @@
+package tech.relaycorp.courier.data.network
+
+import kotlinx.coroutines.flow.Flow
+import java.io.InputStream
+
+interface CogRPC {
+
+    fun deliverCargo(address: String, cargoes: List<MessageDelivery>): Flow<MessageDeliveryAck>
+
+    fun collectCargo(address: String, cca: MessageDelivery): Flow<MessageReceived>
```

Comment:
Similarly, the signature of this function is likely to be:

```kotlin
@Throws(CogRPCTimeoutException::class)
fun collectCargo(collectionRequest: CargoCollectionRequest): Flow<OutputStream>
```

Where:

```kotlin
// This class will come from the core library (not relaynet-cogrpc-jvm)
data class CargoCollectionRequest(
    val localId: String,
    val cca: InputStream
)
```

Where the "local id" would be analogous to the one in a `CargoDeliveryRequest` as described above. 

As I'm writing this, I wonder whether we should just combine the two data classes into one.

Note that each `OutputStream` output by this function represents a cargo.